typings/index.d.ts

```diff
@@ -165,12 +165,14 @@ declare namespace commander {
      *
      * @param {string} flags
      * @param {string} [description]
-     * @param {((arg1: any, arg2: any) => void) | RegExp} [fn] function or default
+     * @param {((value: string, previous: any) => void) | RegExp} [fn] function or default
      * @param {*} [defaultValue]
      * @returns {Command} for chaining
      */
-    option(flags: string, description?: string, fn?: ((arg1: any, arg2: any) => void) | RegExp, defaultValue?: any): Command;
     option(flags: string, description?: string, defaultValue?: any): Command;
```

Comment:
I think this overload should come last, otherwise it matches any call that does not have a default value because of the any on the third parameter. Reference: https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#function-overloads

`option(flags: string, description?: string, defaultValue?: any): Command;`